CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## edge-19.1.3
+
+* Controller
+  * Improved API for `ListPods` (thanks @alenkacz!)
+  * Fixed `GetProfiles` API call not returning immediately when no profile
+    exists (resulting in proxies logging warnings)
+* Web UI
+  * Improved resource detail pages now show all resource types
+  * Fixed stats not appearing for routes that have service profiles installed
+* CLI
+  * Added `linkerd install-sp` command to install service profiles into the
```

Comment:
How about "Added optional `linkerd install-sp` command to generate service profiles for the control plane, providing per-route metrics for control plane components"?